docs/sql-reference/sql-statements/data-definition/ALTER TABLE.md

```diff
@@ -112,22 +112,25 @@ DROP TEMPORARY PARTITION <partition_name>
 
 #### Modify partition properties
 
-Syntax:
+**Syntax**
 
 ```sql
 ALTER TABLE [database.]table
-MODIFY PARTITION p1|(p1[, p2, ...]) SET ("key" = "value", ...)
+    MODIFY PARTITION { <partition_name> | partition_name_list | (*) }
+        SET ("key" = "value", ...);
 ```
 
-Note:
+**Usages**
+
+- The following properties of a partition can be modified:
 
-1. The following properties of a partition can be modified:
+  - storage_medium
+  - storage_cooldown_ttl or storage_cooldown_time
+  - replication_num
 
-   - storage_medium
-   - storage_cooldown_time
-   - replication_num
+- For the table has only one partition, partition name is the same as the table name. If the table is divided into multiple partitions, you can use `(*)`to modify the properties of all partitions. which is more convenient.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/986ff899-df0c-4c28-a682-3c769ed912a7)
-> the partition name